### ISSUE TITLE
Crash when adding Capability.Photos() condition

### ISSUE THREAD

**Issue by ffittschen** [danthorpe/Operations#280]
Hi,
when I add a condition to an operation using `addCondition(AuthorizedFor(Capability.Photos()))` I always get a crash with this error message:
```
2016-04-14 17:51:28.288 SomeApp[63049:15884898] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: '-[PHPhotoLibrary init] unsupported'
*** First throw call stack:
(
	0   CoreFoundation                      0x000000010c9d8d85 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x000000010f66adeb objc_exception_throw + 48
	2   Photos                              0x000000010d8fd8b9 -[PHPhotoLibrary initSharedLibrary] + 0
	3   Operations                          0x000000010c56cff7 _TTSf4g_n___TFC10Operations16PhotosCapability34requestAuthorizationWithCompletionfbT_T_T_ + 231
	4   Operations                          0x000000010c56cabc _TTWC10Operations16PhotosCapabilityS_14CapabilityTypeS_FS1_34requestAuthorizationWithCompletionfbT_T_T_ + 76
	5   Operations                          0x000000010c5deb5b _TFC10Operations9Authorize7executefT_T_ + 315
	6   Operations                          0x000000010c553357 _TFC10Operations9Operation4mainfT_T_ + 1607
	7   Operations                          0x000000010c5533bd _TToFC10Operations9Operation5startfT_T_ + 77
	8   Foundation                          0x000000010d438336 __NSOQSchedule_f + 194
	9   libdispatch.dylib                   0x000000011084d3eb _dispatch_client_callout + 8
	10  libdispatch.dylib                   0x000000011083382c _dispatch_queue_drain + 2215
	11  libdispatch.dylib                   0x0000000110832d4d _dispatch_queue_invoke + 601
	12  libdispatch.dylib                   0x0000000110835996 _dispatch_root_queue_drain + 1420
	13  libdispatch.dylib                   0x0000000110835405 _dispatch_worker_thread3 + 111
	14  libsystem_pthread.dylib             0x0000000110b8a4de _pthread_wqthread + 1129
	15  libsystem_pthread.dylib             0x0000000110b88341 start_wqthread + 13
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

Shouldn't [this line](https://github.com/danthorpe/Operations/blob/development/Sources/Features/iOS/PhotosCapability.swift#L78) use `PHPhotoLibrary.sharedPhotoLibrary()` instead of `PHPhotoLibrary()`?

**Comment by danthorpe** [comment]
Oh! Great catch - will look into this, and why it's not getting caught by the unit tests...

**Comment by danthorpe** [comment]
Hi @ffittschen - this is fixed, in `development` branch. It will get released as part of version 2.10.0 early next week I expect.

In the meantime, you can use `development` in your own project, either by pulling origin into your fork, or pointing to `pod 'Operations/+Photos', :git => 'https://github.com/danthorpe/operations', :branch => 'development'` in your Podfile.

Thanks for reporting this - great spot - don't think I would have caught it, as I don't use `PHPhotosLibrary` - but then you maybe be the first!
